src/librustc_driver/lib.rs

```diff
@@ -65,8 +65,8 @@ pub const EXIT_SUCCESS: i32 = 0;
 /// Exit status code used for compilation failures and invalid flags.
 pub const EXIT_FAILURE: i32 = 1;
 
-const BUG_REPORT_URL: &str = "https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.\
-                              md#bug-reports";
+const BUG_REPORT_URL: &str = "https://github.com/rust-lang/rust/issues/new\
+    ?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md";
```

Comment:
Hm this looks like a pretty annoying URL to output. I'm of half a mind to say we should create a redirect at, say, rust-lang.org/compiler-bug-report so we can readily update things in the future. It seems fine for the time being though.

Reply:
That's a good idea